### DISCUSSION TITLE
How to compress finetuned model?

### DISCUSSION THREAD

**Discussion by greek12man** [erew123/alltalk_tts#217] [Q&A] [answered]
I made various finetuned models with google colab (my machine was not able to handle it) and it works beautifully. 
The only issue is that the model.pth file is 5.22 GB. If only i could compress it, I could have more finetuned models.
The only info i found was here:
https://github.com/coqui-ai/TTS/discussions/3474

by a familiar name :D
however, i am just a user of the program, and have no clue how to use or implement this code.

**Comment by erew123** [comment, marked as answer]
Have a look in the zip of AllTalk 1.8 in the finetune folder. There is a script in there. 

https://github.com/erew123/alltalk_tts/releases/tag/1.8

Thanks

**Reply by greek12man** [reply]
Thank you. I put the script compact.py inside the folder alltalk_tts\finetune\put-voice-samples-in-here
then i put the model i want to compress, renamed to best_model.pth
used cmd on the folder, and run the command
python compact.py
